apps/reactotron-app/src/renderer/pages/home/welcome.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { shell } from "electron"
+import styled from "styled-components"
+import { reactotronLogo } from "../../images"
+import { EmptyState } from "reactotron-core-ui"
+
+const WelcomeText = styled.div`
+  font-size: 1.25em;
+  margin-bottom: 5px;
+`
+
+const Container = styled.div`
+  display: flex;
+  padding: 4px 8px;
+  margin-top: 20px;
+  border-radius: 4px;
+  cursor: pointer;
+  background-color: ${(props) => props.theme.backgroundLighter};
+  color: ${(props) => props.theme.foreground};
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+`
+
+function openDocs() {
+  shell.openExternal("https://github.com/infinitered/reactotron#documentation")
```

Comment:
```suggestion
  shell.openExternal("https://github.com/infinitered/reactotron")
```

The beta and the master branch READMEs have different headings, so let's change this to be just a link to the repo to be safe.